core/src/main/java/org/apache/druid/java/util/common/io/Closer.java

```diff
@@ -123,6 +123,17 @@ private Closer()
     return closeable;
   }
 
+  /**
+   * Registers a list of  {@code closeable} to be closed when this {@code Closer} is
+   * {@linkplain #close closed}.
+   */
+  // close. this word no longer has any meaning to me.
+  public <C extends Closeable> Iterable<C> register(Iterable<C> closeables)
```

Comment:
Some class may implement both `Iterable` and `Closeable`, so ambiguity with the other `register()` method is possible. I suggest to call this method `registerAll()`.

Reply:
Good suggestion, done!